### DISCUSSION TITLE
What is difference between deafult_pool and parallelism?

### DISCUSSION THREAD

**Discussion by developingbear** [apache/airflow#26147] [Q&A] [answered]
What is difference between deafult_pool and parallelism?

My guess is that deafult_pool and parallelism equally limit the maximum number of tasks. 
Are there any differences I'm not aware of?

**Comment by potiuk** [comment, marked as answer]
Several:
* parallelism is tasks per scheduler (there can be more than one scvheduler) and default pool is total number of tasks (no matter how many schedulers)
* you can add more pools and default pool only applies to tasks that are not assigned to a specific pool

They are only equivalent if you have 1 scheduler AND no tasks assigned to other pools. But in all other circumstances, they are different.

**Reply by developingbear** [reply]
thank you!!
